scripts/merge-and-build.sh

```diff
@@ -240,6 +240,32 @@ echo "=========="
 ssh ocp-build@rcm-guest.app.eng.bos.redhat.com " /mnt/rcm-guest/puddles/RHAOS/scripts/push-to-mirrors-bot.sh simple ${OSE_VERSION}"
 
 echo
+echo "=========="
+echo "Publish the oc binary
+echo "=========="
+ssh ocp-build@rcm-guest.app.eng.bos.redhat.com \
+    sh -s "$OSE_VERSION" "${VERSION#v}" <<-'EOF'
+	set -eux
+	rpm=/mnt/rcm-guest/puddles/RHAOS/AtomicOpenShift/$1/latest/x86_64/os
+	rpm=$rpm/Packages/atomic-openshift-clients-redistributable-$2
+	rpm=$(echo "$rpm"*)
+	tmpdir=$(mktemp -dt ocbinary.XXXXXXXXXX)
+	trap "rm -rf '$tmpdir'" EXIT INT TERM
+	mkdir -p "$tmpdir/$2/"{linux,macosx,windows}
+	rpm2cpio "$rpm" | cpio -D "$tmpdir" -idm --quiet
```

Comment:
I didn't know, it's a relatively new option (committed in 2010, released in
2.12, 2015) not yet present on the version we have there (2.10).  No big deal,
just added an extra `cd`.